### PR TITLE
Release notes for 2.4.13

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+- :jirabug:`1548`: |Percona Xtrabackup| enables updating the
+  ib_buffer_pool file with the latest pages present in the buffer pool by
+  setting the ``innodb_buffer_pool_dump_now`` variable to `ON` .
+- :jirabug:`1609`: The ``--version_check`` option can be disabled at build time
+
+Bus Fixed
+================================================================================
+
+- :jirabug:`1536`: |Percona Xtrabackup| 2.4.11, MySQL 5.7.21-21-log Percona Server partial backup and errors in MySQL log
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: xtrabackup did not delete missing tables from partial backup which led to error messages logged by server on startup

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- :jirabug:`1569`: The ``--history`` option did not work when autocommit was disabled
+- :jirabug:`1570`: |Percona XtraBackup| was affected by issue `PS-4519 <https://jira.percona.com/browse/PS-4519>`_ (Percona Server for MySQL could not be compiled on CentOS 6 with gcc 6.3.1)
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Should go to "other bugs"

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- :jirabug:`1648`: ``xtrabackup`` could fail to open tablespace with the error "Encryption information in datafile: <NAME.ibd> can't be decrypted"
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: xtrabackup could fail to backup encrypted tablespace when it was recently created or atlered

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+- :jirabug:`1668`: When the ``--throttle`` option was used, the applied value was different from the one specified by the user (off by one error)
+- :jirabug:`1672`: It was not allowed for MTS (multi-threaded slaves)  without GTID to be backed up with ``--safe-slave-backup``
+- :jirabug:`1679`: |Percona Xtrabackup| could crash when the ``ALTER TABLE … TRUNCATE PARTITION`` command was run during a backup without locking DDL
+- :jirabug:`1683`: ``xbcrypt`` could display an assertion failure and generated core if the required parameters are missing
+- :jirabug:`1691`: The "out of memory" error was reported when making a backup with ``--lock-ddl-per-table``
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: can we combine 1691 and 1698 into one with description "Using --lock-ddl-per-table caused server to scan all records of partitioned tables which could lead to out of memory error"

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+- :jirabug:`1698`: |Percona Xtrabackup| scanned all rows of partitioned tables when using ``--lock-ddl-per-table``
+- :jirabug:`1704`: `xtrabackup --prepare` could hang while performing insert buffer merge
+- :jirabug:`1711`: Incremental backups did not create or update ``xtrabackup_binlog_info``
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Incremental backups did not update xtrabackup_binlog_info with --binlog-info=lockless

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+|date|. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.13/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+Improvements and New Features
+================================================================================
+
+- :jirabug:`1548`: |Percona Xtrabackup| enables updating the
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst:15]: Can we update https://www.percona.com/doc/percona-xtrabackup/2.4/xtrabackup_bin/lru_dump.html with new options, rename it to "buffer pool dump" and reference it from RN? Also xtrabackup option reference should be updated. And `innodb_buffer_pool_dump_now` became `--dump-innodb-buffer-pool`

**Review by gl-sergei — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  ib_buffer_pool file with the latest pages present in the buffer pool by
+  setting the ``innodb_buffer_pool_dump_now`` variable to `ON` .
+- :jirabug:`1609`: The ``--version_check`` option can be disabled at build time
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Should go to "bugs fixed", not much of a feature

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 6 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst:15]: This update is tracked by https://jira.percona.com/browse/PXB-1736 I've added a note to that ticket. I will make another PR for this update which will contain a reference to documentation from the release notes. The documentation update will be available from the site at the same time as the release notes.

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+- :jirabug:`1683`: ``xbcrypt`` could display an assertion failure and generated core if the required parameters are missing
+- :jirabug:`1691`: The "out of memory" error was reported when making a backup with ``--lock-ddl-per-table``
+- :jirabug:`1698`: |Percona Xtrabackup| scanned all rows of partitioned tables when using ``--lock-ddl-per-table``
+- :jirabug:`1704`: `xtrabackup --prepare` could hang while performing insert buffer merge
+- :jirabug:`1711`: Incremental backups did not create or update ``xtrabackup_binlog_info``
+
+Other bugs fixed: :jirabug:`1632`
+
+.. |release| replace:: 2.4.13
+.. |date| replace:: January 17, 2018
```
Comment (hrvojem) [inline storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: release date is Jan *18th*

Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Fixed

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 7 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Updated as requested

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 5 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Updated as requested

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 4 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Updated as requested

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 3 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Updated as requested

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 2 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst]: Updated as requested

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 6 above)
Comment (gl-sergei) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst:15]: Still correct parameter name is `--dump-innodb-buffer-pool`. Please also thank Marcelo Altmann (@altmannmarcelo ) for contribution.

**Review by gl-sergei — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst (hunk 6 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.13.rst:15]: Updated as requested

**Review by gl-sergei — APPROVED** [review]

**Pull request merged**